src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Everyone disperse, the truck started work!!!");
```

Comment:
The message printed by the `doWork` method should be more informative and indicate that the 'Truck' is the type of machine working. Consider rephrasing it to include the machine type.